### ISSUE TITLE
Inclusion of ALL page versions (including "self") in hreflang

### ISSUE THREAD

**Issue by dleigh** [oferwald/transposh#14]
Hello Ofer,
According to this page: https://developers.google.com/search/docs/specialty/international/localized-versions and other things I'm seeing, the hreflang tag should also exist for the current page and not only for the OTHER language versions.  So with 3, languages: nl, en, vi for example, every page that is translated (let's assume all of them) should have 3 hreflang tags.  

Apparently it was Google that invented the hreflang tag in 2011, so I'll assume that they are the ones to establish the correct practice.  Consequently, it would appear that the Transposh implementation is incomplete or in error on this point. 

There is also the idea of the "x-default" hreflang tag.  Its inclusion appears to be optional, and its purpose is a bit more subtle (and confusing), but it seems like there should be an option to create it.
Thanks

**Comment by dleigh** [comment]
In transposh.php, this appears to be the modification necessary to include a "self" hreflang tag: 

`
    /**
     * Implements - http://googlewebmastercentral.blogspot.com/2010/09/unifying-content-under-multilingual.html
     */
    function add_rel_alternate() {
        if (is_404()) {
            return;
        }
        $widget_args = $this->widget->create_widget_args($this->get_clean_url());
        tp_logger($widget_args, 4);
        foreach ($widget_args as $lang) {
//            if (!$lang['active']) {   <<<<<<<<<<<<<<<<<<<<<<<<< comment out
                echo '<link rel="alternate" hreflang="' . $lang['isocode'] . '" href="';
                    if ($this->options->full_rel_alternate) {
                        $current_url = ( is_ssl() ? 'https://' : 'http://' ) . transposh_utils::get_clean_server_var('HTTP_HOST') . transposh_utils::get_clean_server_var('REQUEST_URI');
                        $url = transposh_utils::rewrite_url_lang_param($current_url, $this->home_url, $this->enable_permalinks_rewrite, $lang['isocode'], $this->edit_mode);
                        if ($this->options->is_default_language($lang['isocode'])) {
                            $url = transposh_utils::cleanup_url($url, $this->home_url);
                        }
                        echo $url;
                    } else {
                        echo $lang['url'];
                    }

                echo '"/>';
//            }      <<<<<<<<<<<<<<<<<<<<<<<<<<<<<< comment out
        }
    }


`
Basically just commenting out the **if statement** checking to see if it's the "active" page and including it instead of excluding it. If we want to change the way it functions to ALWAYS include the "self" hreflang tag it's pretty straightforward.  If it needs to be optional (which makes sense for backwards compatibility with how Transposh has always functioned) then, of course it's a bit more effort to make it a parameter.

**Comment by oferwald** [comment]
As far as I remember the google compatibility setting should have fixed that, and I also think Google here is being stupid. Why link to yourself, and with an FQDN no less.
